core/spec/lib/spree/core/importer/order_spec.rb

```diff
@@ -403,86 +403,96 @@ module Core
-        order = Importer::Order.import(user,params)
+        params = { payments_attributes: [{ amount: '4.99',
+                                           payment_method: payment_method.name,
+                                           state: 'completed' }] }
+        order = Importer::Order.import(user, params)
         expect(order.payments.first.amount).to eq 4.99
       end
 
       it 'builds a payment using status as fallback' do
-        params = { :payments_attributes => [{ amount: '4.99',
-                                              payment_method: payment_method.name,
-                                              status: 'completed' }] }
-        order = Importer::Order.import(user,params)
+        params = { payments_attributes: [{ amount: '4.99',
+                                           payment_method: payment_method.name,
+                                           status: 'completed' }] }
+        order = Importer::Order.import(user, params)
         expect(order.payments.first.amount).to eq 4.99
       end
 
       it 'handles payment building exceptions' do
-        params = { :payments_attributes => [{ amount: '4.99',
-                                              payment_method: 'XXX' }] }
+        params = { payments_attributes: [{ amount: '4.99',
+                                           payment_method: 'XXX' }] }
         expect {
           order = Importer::Order.import(user, params)
         }.to raise_error /XXX/
       end
 
       it 'build a source payment using years and month' do
-        params = { :payments_attributes => [{
-                                              amount: '4.99',
-                                              payment_method: payment_method.name,
-                                              status: 'completed',
-                                              source: {
-                                                name: 'Fox',
-                                                last_digits: "7424",
-                                                cc_type: "visa",
-                                                year: '2022',
-                                                month: "5"
-                                              }
-                                            }]}
+        params = { payments_attributes: [{
+                                           amount: '4.99',
+                                           payment_method: payment_method.name,
+                                           status: 'completed',
+                                           source: {
+                                             name: 'Fox',
+                                             last_digits: "7424",
+                                             cc_type: "visa",
+                                             year: '2022',
+                                             month: "5"
+                                           }
+                                         }]}
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.<br>Space inside } missing.